lib/js/hideAnswer.js

```diff
@@ -0,0 +1,7 @@
+const answer = document.querySelector('[data-js="card__answer"]');
+let button = document.querySelector('[data-js="card__button--answer"]');
+
+button.addEventListener("click", () => {
+  answer.classList.toggle("card__p--hidden");
+  console.log("button");
```

Comment:
Here you have a tiny `console.log` here. Everytime you click that button the console.log is shown in the console.
Highly recommend not using `console.log` in production code. ;-)